src/estimators.jl

```diff
@@ -371,14 +366,14 @@ end
         max_cat_threshold = 32,
         cat_l2 = 10.0,
         cat_smooth = 10.0,
-        metric = [\"multi_logloss\"],
+        metric = [""],
```

Comment:
Default for metric is an empty string `""` not a vector but this applies to one supported metric, or metrics introduced as a string such as `metric=binary_logloss` whereas for most use cases with multiple metrics we need a vector of strings, hence leaving it here.